zcash_client_sqlite/src/testing.rs

```diff
@@ -815,6 +841,79 @@ impl TestFvk for DiversifiableFullViewingKey {
     }
 }
 
+#[cfg(feature = "orchard")]
+impl TestFvk for orchard::keys::FullViewingKey {
+    type Nullifier = orchard::note::Nullifier;
+
+    fn sapling_ovk(&self) -> Option<sapling::keys::OutgoingViewingKey> {
+        None
+    }
+
+    fn orchard_ovk(&self, scope: zip32::Scope) -> Option<orchard::keys::OutgoingViewingKey> {
```

Comment:
We probably only need external scope; for internal scope we will always decrypt with the ivk?

Reply:
I didn't write `TestFvk` with the intention of it being used directly in tests; it provides the functionality needed by `TestState`.

Separately, you do not want to use external ovks for internal change, otherwise giving out an external ivk ovk pair leaks internal wallet actions.